docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 6
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, users can configure IP pool information and serve IP addresses to virtual machines running on Harvester clusters via DHCP.
+
+This is accomplished using the vm-dhcp-controller add-on.
+
+## Install and Enable The vm-dhcp-controller Add-on
+
+The vm-dhcp-controller add-on is not packaged with Harvester, but it can be found in the [expreimental-addon repo](https://github.com/harvester/experimental-addons). Users need to install the add-on first:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/harvester/experimental-addons/main/harvester-vm-dhcp-controller/harvester-vm-dhcp-controller.yaml
+```
+
+After the add-on is installed, enable the add-on on the dashboard.
```

Comment:
After the add-on is installed, enable the add-on on the dashboard.
->
After this add-on is installed, user can enable it from the Harvester dashboard or `kubeclt`.